lib/url_parser.rb

```diff
@@ -162,7 +162,7 @@ def includes_domain?
   def extractable_early?
     return false if website_url?
 
-    match = url.match(/([\w\.@\:\-_~]+)\.#{domain_regex}\/([\w\.@\:\-\_\~]+)/i)
+    match = url.match(/([\w\.\@\:\-\~]+)\.#{domain_regex}\/([\w\.\@\:\-\~]+)/i)
```

Comment:
2 changes:
* removed the `_` because it's included in `\w`
* some of the characters were escaped and some were'nt, so I just added backslashes to escape them all for consistency